flixel/FlxGame.hx

```diff
@@ -758,8 +759,6 @@ class FlxGame extends Sprite
 		}
 		FlxArrayUtil.clearArray(FlxG.swipes);
 		#end
-		
-		filters = filtersEnabled ? _filters : null;
```

Comment:
I think there was a reason why this happens every frame.. I think the contents of the array can change without a `setFilters()` call, and that wouldn't trigger OpenFL's setter.